### ISSUE TITLE
Create new simplified shell template

### ISSUE THREAD

**Issue by gentzkow** [gentzkow/template#96]

**Comment by gentzkow** [comment]
PRINCIPLES
- Someone who knows nothing about template can run scripts from the directory they live in as usual
- Scripts contain nothing that looks unusual to a user who doesn't know the template structure
- Scripts don't need to know about any config variables outside their own module

TO DO
- Revisit dependency handling for Python, R, Stata
- Path handling for repo graph; scripts report their dependencies?
- Creating /external/ with links
- Copying inputs for /paper_slides/
- Using .yaml for local_env?
- Defaults for executables
- Make local_env totally optional?
- Write nice header comments for shell commands
- check_setup command
- What about output_local / external output?
- How to separate tables/figures from main scripts
- Clean up make.log -- fix run_stata, run_r, and run_python commands
- License 
- Add to lab manual recommended tools -- VS Code?
- Update so individual make.sh scripts can run out of the box without any setup step; need to handle case where local_env.sh doesn't exist; use default version

**Comment by gentzkow** [comment]
TO DO
* Test LaTex template to make sure figure / table / reference hyperlinks are all compiling correctly

**Comment by gentzkow** [comment]
@arjunsrini Thanks for checking. I've been doing some more work on this. I'm going to finish those changes and wrap up this issue. I'll then open some new issues for you and @shrishj to move on to next steps.

**Comment by gentzkow** [comment]
@arjunsrini @shrishj Sorry for the slow reply here. I'd been hoping to make more progress last weekend but I got sidetracked by some other things.

The most recent iteration I've committed here is very much work in progress. It is definitely not yet ready for prime time, and I'm therefore reluctant to merge to `master`. What do you think about making this a new fork of `template`? Or, alternatively, we could just make it a clean new repository?

While we figure that out, I think it would be worth both of you cloning the current version on this branch, reading through the `README`, and trying to setup + run it. One discrepancy to flag is that the `README` still refers to a separate `get_inputs.sh` script but I've so far folded that into `make.sh`.

**Comment by shrishj** [comment]
Hi @gentzkow, Arjun and I have managed to clone the branch and get it to work! One small thing in the README is that it says to use `bash make.sh` when I think that it should be `bash run_all.sh`. We had a short discussion regarding the forked vs new repository, and we were leaning towards making a new repository?

**Comment by gentzkow** [comment]
Great. Thanks @shrishj.

I created a new repo called `GentzkowLabTemplate`. It is private for now. Once we're ready to release it we'll make it public and at the same time change the name of this repo to `template-archived`.

I would ask that you do the following:

1. Make your proposed edit to the readme
2. Open a new issue in `GentzkowLabTemplate` to import the content from this branch
3. Port over the content of this branch and merge to the main branch of `GentzkowLabTemplate`
4. Close this issue with a summary comment that points to that revision of `GentzkowLabTemplate`
5. Add a page to the `GentzkowLabTemplate` wiki called "Roadmap" and add to it the to dos that we've noted in the thread above

Once this is done I'll open some new issues in `GentzkowLabTemplate` to track our next steps.

**Comment by shrishj** [comment]
### Summary:

This issue looked to create a new simplified shell template. We will continue to develop the shell template in [GentzkowLabTemplate](https://github.com/gentzkow/GentzkowLabTemplate/tree/1132ed7cd7d3f978755be4925ee9fdee909c2cd0). The contents of this [branch](https://github.com/gentzkow/template/tree/f0de56f0f967f4d1b80cd32fb9bf6411626889e4) have been transferred to the new repository to maintain progress. 

Final state of branch can be found [here](https://github.com/gentzkow/template/tree/f0de56f0f967f4d1b80cd32fb9bf6411626889e4)

cc @gentzkow
